daprdocs/content/en/reference/components-reference/supported-pubsub/setup-solace-amqp.md

```diff
@@ -0,0 +1,108 @@
+---
+type: docs
+title: "Solace-AMQP"
+linkTitle: "Solace-AMQP"
+description: "Detailed documentation on the Solace-AMQP pubsub component"
+aliases:
+  - "/operations/components/setup-pubsub/supported-pubsub/setup-solace-amqp/"
+---
+
+## Component format
+
+To setup Solace-AMQP pubsub create a component of type `pubsub.solace.amqp`. See [this guide]({{< ref "howto-publish-subscribe.md#step-1-setup-the-pubsub-component" >}}) on how to create and apply a pubsub configuration
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: solace
+spec:
+  type: pubsub.solace.amqp
+  version: v1
+  metadata:
+    - name: url
+      value: 'amqp://localhost:5672'
+    - name: username
+      value: 'default'
+    - name: password
+      value: 'default'
+```
+
+{{% alert title="Warning" color="warning" %}}
+The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
+{{% /alert %}}
+
+## Spec metadata fields
+
+| Field              | Required | Details | Example |
+|--------------------|:--------:|---------|---------|
+| url    | Y  | Address of the AMQP broker. Can be `secretKeyRef` to use a secret reference. <br> Use the **`amqp://`** URI scheme for non-TLS communication. <br> Use the **`amqps://`** URI scheme for TLS communication. | `"amqp://host.domain[:port]"`
+| username | Y | The user name to connect to the broker with | default
+| password | Y | The password to connect to the broker with | default
+| anonymous | N | To connect to the broker without credential validation. Only works if enabled | true
```

Comment:
I presume that if this is set to `true` then you do not need username and password, is that correct. Either way can you make a comment here on how to use `anonymous` setting, correct and when?